packages/backend/api/src/models/post-like-model.ts

```diff
@@ -0,0 +1,72 @@
+import { jsonObjectFrom } from 'kysely/helpers/mysql';
+
+import type { QueryError } from '@app/backend-shared';
+import { db } from '@app/backend-shared';
+
+export default {
+  // GET **************************************************
+  getPostsLikesCountByPost(postId: number) {
+    return db
+      .selectFrom('post_like')
+      .select(({ fn }) => [fn.count<number>('post_id').as('like_count')])
+      .where('post_id', '=', postId)
+      .executeTakeFirst();
+  },
+
+  getPostsLikesCountByGiver(userId: number) {
+    return db
+      .selectFrom('post_like')
+      .select(({ fn }) => [fn.count<number>('user_id').as('like_count')])
+      .where('user_id', '=', userId)
+      .executeTakeFirst();
+  },
+
+  getPostsLikesCountByReceiver(userId: number) {
+    return db
+      .selectFrom('post_like')
+      .select(({ fn }) => [fn.count<number>('post_id').as('like_count')])
+      .innerJoin('post', (join) =>
+        join.onRef('post.id', '=', 'post_like.post_id'),
+      )
```

Comment:
Normalement ceci devrait suffire
```suggestion
      .innerJoin('post', 'post.id', 'post_like.post_id')
```